langchain/tools/graphql/tool.py

```diff
@@ -0,0 +1,51 @@
+import json
+from typing import Optional
+
+from langchain.callbacks.manager import (
+    AsyncCallbackManagerForToolRun,
+    CallbackManagerForToolRun,
+)
+from langchain.tools.base import BaseTool
+from langchain.utilities.graphql import GraphQLAPIWrapper
+
+
+class BaseGraphQLTool(BaseTool):
+    """Base tool for querying a GraphQL API."""
+
+    graphql_wrapper: GraphQLAPIWrapper
+
+    name = "query_graphql"
+    description = """
```

Comment:
maybe intentional but currently there's a newline before and after all the text. could get rid of that without changing formatting with 
```python
"""\
paragraph1
paragraph2\
"""